ui/scripts/instances.js

```diff
@@ -618,11 +618,11 @@
                         createForm: {
                             title: 'label.action.destroy.instance',
                             desc: 'label.action.destroy.instance',
-                isWarning: true,
+                            isWarning: true,
                             preFilter: function(args) {
-                                if (isAdmin() || isDomainAdmin()) {
-                                    args.$form.find('.form-item[rel=expunge]').css('display', 'inline-block');
-                                } else {
+                                // Hide the expunge checkbox when the authenticated user
+                                // can't expunge VMs. Related to CLOUDSTACK-8580.
```

Comment:
I think the var name is explanatory. the comment could go with allocation and not use of the thingy.